Gemfile

```diff
@@ -0,0 +1,73 @@
+source 'https://rubygems.org'
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+ruby '3.0.5'
+
+# Bundle edge Rails instead: gem "rails", github: "rails/rails", branch: "main"
+gem 'rails', '~> 7.0.4'
+
+# The original asset pipeline for Rails [https://github.com/rails/sprockets-rails]
+gem 'sprockets-rails'
+
+# Use postgresql as the database for Active Record
+gem 'pg', '~> 1.1'
+
```

Comment:
- [ ] I'm not getting greetings from the back end to the front end because of `Cors-Policy` issue, Please fix this issue.
  - To fix this issue install and set up [rack-cors gem](https://github.com/cyu/rack-cors) in your project.

![image](https://user-images.githubusercontent.com/83803414/208503214-3bbf561b-ba39-4588-bdd1-673d4cf4feb5.png)